templates/habootstrap-formula/cluster_resources_nfs.j2

```diff
@@ -1,6 +1,6 @@
 {%- set data = pillar.cluster.configure.template.parameters %}
 {%- set drbd = salt['pillar.get']('drbd', default={}, merge=True) %}
-{%- set nfsid = 1 %}
+{%- set nfsid = 10 %}
```

Comment:
Just for curiosity, any reason change the id from `1` to `10`?

Reply:
I had an overlap somewhere ... Does not really matter.